src/main/webapp/i18n/de/student-dashboard.json

```diff
@@ -42,7 +42,8 @@
                 "noDate": "Ohne Datum",
                 "all": "Übungsgruppen",
                 "registered": "Meine Übungsgruppen",
-                "further": "Weitere Übungsgruppen"
+                "further": "Weitere Übungsgruppen",
+                "noUpcomingSession": " Kein Termin"
```

Comment:
Is the space intended? I personally think that we should add this to the html-file and not the translation, what do you think?